src/components/ebay-breadcrumb/README.md

```diff
@@ -0,0 +1,40 @@
+# ebay-breadcrumb
+
+## ebay-breadcrumb Usage
+### Last item as the parent
+```marko
+<ebay-breadcrumb aria-label='You are here'>
+    <ebay-breadcrumb-item href='https://...'>eBay</ebay-breadcrumb-item>
+    <ebay-breadcrumb-item href='https://...'>Auto Parts and Vehicles</ebay-breadcrumb-item>
+    <ebay-breadcrumb-item href='https://...' _sp='' navsrc='{}' navsrc='{}'>Motors Parts and Accessories</ebay-breadcrumb-item>
+</ebay-breadcrumb>
+```
+### Last item as the current
+```marko
+<ebay-breadcrumb aria-label='You are here'>
```

Comment:
Is `You are here` a typical example? Let's pick text that's realistic for the use case (across all examples).